jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -233,6 +233,12 @@ def _get_zoom_limits(self, image):
                                     (self.state.x_min, self.state.y_max),
                                     (self.state.x_max, self.state.y_max),
                                     (self.state.x_max, self.state.y_min)))
+
+        # Disable if there is canvas rotation.
```

Comment:
Ah, so you did end up disabling the zoom box in Compass? Should make a note in the plugin user doc that this will not show if canvas is rotated.

Reply:
Hmmm.... I think you added this code.  Do you think now that we just rotate the compass image entirely that this can safely be removed?

Reply:
I removed this code and clarified the behavior in the docs for both plugins.  Let me know what you think!

Reply:
It works better than I thought it would! 😸